pimpmycause/profiles/views.py

```diff
@@ -67,6 +67,9 @@ def homepage(request):
 
     return render(request, 'index.html', context)
 
+def profile_detail(request, slug):
+    person = PimpUser.objects.get(slug=slug)
+    return render(request, 'detail.html', {'person':person})
```

Comment:
the template is missing the full path it should be `profile/detail.html` so it is relative to where it is in the templates forlder